src/routes/dashboard/+page.svelte

```diff
@@ -1,8 +1,20 @@
 <script>
-	import { authStore } from '$lib/store/store';
+	import Card from '$components/Card.svelte';
+
+	const tricks = Array.from({ length: 20 }).map((_, i) => {
```

Comment:
Nice! I didn't know about this method to easily create arrays with mock data. So convenient